ols/app/models/models.py

```diff
@@ -342,6 +342,35 @@ class HealthResponse(BaseModel):
     }
 
 
+class ReadinessResponse(BaseModel):
+    """Model representing a response to a readiness request.
+
+    Attributes:
+        ready: The readiness of the service.
+        reason: The reason for the readiness.
```

Comment:
@onmete do we want the reason field ? just thinking if we can keep in sync with liveness, unless there is a use for this.

Reply:
Telling why service is not ready sounds like a good reason to me. But I didn't find any required schema for these two endpoints, so I guess it is on our needs. Do you have concerns with this schema?